### ISSUE TITLE
Unable to install grails melody plugin

### ISSUE THREAD

**Issue by GoogleCodeExporter** [javamelody/javamelody#200]
```
What steps will reproduce the problem?
1.run the grails install plugin command "grails install-plugin grails-melody"


What is the expected output? What do you see instead?
The installation should complete after some time, but it hangs. Below is the 
output of the command.

pradeep@pradeep:~/Amphisoft/amphihub/ekluvya$ grails install-plugin 
grails-melody
Welcome to Grails 1.3.7 - http://grails.org/
Licensed under Apache Standard License 2.0
Grails home is set to: /home/pradeep/tools/grails-1.3.7

Base Directory: /home/pradeep/Amphisoft/amphihub/ekluvya
Resolving dependencies...
Dependencies resolved in 2692ms.
Running script /home/pradeep/tools/grails-1.3.7/scripts/InstallPlugin.groovy
Environment set to development
Resolving plugin grails-melody. Please wait...

Downloading: 
http://svn.codehaus.org/grails-plugins/grails-grails-melody/tags/RELEASE_1_12/gr
ails-grails-melody-1.12.zip ...
Download complete.
Installing zip 
/home/pradeep/.ivy2/cache/org.grails.plugins/grails-melody/zips/grails-melody-1.
12.zip... ...
    [mkdir] Created dir: /home/pradeep/.grails/1.3.7/projects/ekluvya/plugins/grails-melody-1.12
    [unzip] Expanding: /home/pradeep/.ivy2/cache/org.grails.plugins/grails-melody/zips/grails-melody-1.12.zip into /home/pradeep/.grails/1.3.7/projects/ekluvya/plugins/grails-melody-1.12
Installed plugin grails-melody-1.12 to location 
/home/pradeep/.grails/1.3.7/projects/ekluvya/plugins/grails-melody-1.12. ...
Resolving plugin JAR dependencies ...


What version of the product are you using? On what application server, JDK,
operating system?
Ubuntu 11.04

Please provide any additional information below.


```

Original issue reported on code.google.com by `pradeep...@gmail.com` on 16 Mar 2012 at 2:23
- Blocked on: #194, #199
- Merged into: #199
